src/api/orderApi.ts

```diff
@@ -1,13 +1,22 @@
 import RestApi from './restApi';
+import {ApiResponse} from './types';
 
 export interface OrderApi {
-  orderByMarket: (body: any) => Promise<{[key: string]: any}>;
-  orderByPending: (body: any) => Promise<{[key: string]: any}>;
+  orderByMarket: (body: any) => ApiResponse;
+  orderByPending: (body: any) => ApiResponse;
+  fetchAll: () => Promise<any[]>;
 }
 
 export class RestOrderApi extends RestApi implements OrderApi {
-  orderByMarket = (body: any) => this.fireAndForget('/Orders/market', body);
-  orderByPending = (body: any) => this.fireAndForget('/Orders/limit', body);
+  orderByMarket = (body: any) =>
+    this.fireAndForget('/Orders/market', body, {
```

Comment:
but i still believe there should be `post` call, not `fireAndForget`, because errors are possible